content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Quickstart: SCITT for Supply Chains (Preview)"
+description: "Getting Started with SCITT, enabling Software Supply Chain scenarios (Preview)"
+lead: "Push a collection of Supply Chain Statements using SCITT APIs"
+date: 2021-06-09T13:49:35+01:00
+lastmod: 2021-06-09T13:49:35+01:00
+draft: false
+images: []
+menu:
+  developers:
+    parent: "developer-patterns"
+weight: 112
+toc: true
+aliases: 
+  - /docs/developer-patterns/scitt-api/
+---
+
+{{< caution >}}
+The SCITT API is currently in preview and subject to change
+{{< /caution >}}
+
+The **S**upply **C**hain **I**ntegrity, **T**ransparency and **T**rust (SCITT) initiative is a set of [IETF standards](https://datatracker.ietf.org/group/scitt/documents/) for managing the compliance and auditability of goods and services across end-to-end supply chains.
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+To assure insights to supply chain artifacts are current, the SCITT APIs provide a correlation of statements for a specific artifact, allowing newer information to be registered for the most up to date insights.
```

Comment:
Minor: the ability to correlate statements not only allows newer information to help inform decisions, but allows us to see old data as well. 

It allows us to see a full picture of the history of the statements made corresponding to an artefact.

I think its important to mention this.

Reply:
agreed, I've incorporated "the histroy"